web/html/src/manager/state/recurring-states.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from "react";
+import { Messages } from "components/messages";
+import Network from "utils/network";
+import { RecurringStatesDetails } from "./recurring-states-details";
+import { RecurringStatesList } from "./recurring-states-list";
+import { RecurringStatesEdit } from "./recurring-states-edit";
+import { Utils as MessagesUtils } from "components/messages";
+import SpaRenderer from "core/spa/spa-renderer";
+
+/**
+ * See:
```

Comment:
If I'm not mistaken, these are all the files that may render this view and where we might inherit global variables from. Please correct me if I'm wrong.

Reply:
Thanks a lot for pointing that out, I've added that to the list. :)